doc/classes/AnimationNodeStateMachine.xml

```diff
@@ -143,6 +143,7 @@
 			<param index="0" name="name" type="StringName" />
 			<param index="1" name="node" type="AnimationNode" />
 			<description>
+				Replaces the specified animation node with a new animation node.
```

Comment:
```suggestion
				Replaces the given animation node with a new animation node.
```
The methods above use this language so should match

Reply:
Done.